installation/routines/setup_jukebox_core.sh

```diff
@@ -30,6 +30,8 @@ _jukebox_core_install_python_requirements() {
   source "$VIRTUAL_ENV/bin/activate"
 
   pip install --upgrade pip
+  # Remove RPi.GPIO, if still installed - see https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2313
```

Comment:
It's a good idea, but I don't think we need it. Our installation does not qualify for reinstallation anyways. It should not be installed by default.